### DISCUSSION TITLE
Custom tags for sidecar Docker containers

### DISCUSSION THREAD

**Discussion by t-kulmburg** [renovatebot/renovate#11377] [Request Help] [answered]
When running on go, npm and gradle(deepExtract) projects Renovate creates a sidecar Docker container with images from docker.io/renovate/xx. 
Is it somehow possible to configure a fixed tag/version for these images, so that not always the latest is used?

**Comment by viceice** [comment, marked as answer]
https://docs.renovatebot.com/configuration-options/#constraints

**Reply by t-kulmburg** [reply]
thanks!
